tests/conftest.py

```diff
@@ -115,13 +116,18 @@ async def env(env_type: str) -> AsyncGenerator[WorkflowEnvironment, None]:
                 "frontend.workerVersioningDataAPIs=true",
                 "--dynamic-config-value",
                 "system.enableDeploymentVersions=true",
+                "--http-port",
+                str(http_port),
             ],
             dev_server_download_version=DEV_SERVER_DOWNLOAD_VERSION,
         )
     elif env_type == "time-skipping":
         env = await WorkflowEnvironment.start_time_skipping()
     else:
         env = WorkflowEnvironment.from_client(await Client.connect(env_type))
+
+    # TODO(nexus-prerelease): expose this in a principled way
+    env._http_port = http_port  # type: ignore
```

Comment:
Mentioned in other comment, but not sure Python needs to do any Nexus HTTP testing